app/controller.py

```diff
@@ -1,15 +1,19 @@
 import io
 
+import logging
 import numpy as np
 from PIL import Image
 from doc_scanner import run_scan_by_image
 from flask import Blueprint, jsonify, request, send_file
 
+from services.vision_service import VisionService
+
 from app import repository
 
 doc_scanner_bp = Blueprint('doc_scanner', __name__, url_prefix='/doc-scanner')
 ALLOWED_EXTENSIONS = {'png', 'jpg', 'jpeg'}
 
+logging.basicConfig(level=logging.ERROR)
 
 def allowed_file(filename):
```

Comment:
Przeniósłbym to do innego pliku, albo chociaż dał na sam dół, żeby endpointy były bardziej widoczne

Reply:
Jerzy przeniósł, to będzie w następnym PR. @JerzyCode Dobrze pamiętam?